docs/source/overview.rst

```diff
@@ -0,0 +1,94 @@
+Overview
+========
+
+Purpose
+-------
+
+It's easiest to describe The MozDef Event Framework as a set of micro-services you can use to integrate event sources with the `Mozilla Enterprise Defense platform <https://mozdef.readthedocs.io/en/latest/>`_ (MozDef).
+
+Provides
+--------
+
+Many resources are not wholly contained within the datacenter, there are SAAS and IAAS providers which may provide event logging that is external to the network where your SIEM resides. 
+Exposing your security infrastructure doesn't ensure a secure pipeline. In order to solve for this we decided on pulling that information as our method of choice, as opposed to pushing.
+Utilizing cloud based microservices can reduce exposure, ensure efficiency and scalability, and leads to less maintenance.
+
+We created a framework using AWS Cloudformation and the Serverless framework to build a pipeline that will allow placement of lambda scripts specific for an event source and provide continuous deployment and integration.
+This takes the guesswork out of having to write serverless and cloudformation code every time  you want to deploy a pipeline for a new event source and creates a standard method to be used to obtain those events. 
+
+Webhook sources utilize an AWS API gateway that they can post events to. As we scale this out the capabilities will increase.
+The hope is to have a scalable framework that can be deployed for both rest and webhook based sources.
+
+Flow
+----
+
+.. image:: images/MozDef-Event-Framework_process_flow.png
```

Comment:
There is nothing factually wrong with this diagram, but I thought in the past we discussed having separate flow diagrams for each option: CB only, Github only, both with merge.

Reply:
We will still have them, but I haven't created them yet. This is still a WIP.

Reply:
Working on these diagrams now

Reply:
We don't need separate diagrams for each, because the workflow for using a single repo is the same, only the repo source is different. I believe this creates redundancy. 
So we'll only have two diagrams, one for multi-source and one for single source.

We'll just need to advise in the directions any differences.

Reply:
This works for me. Thanks.